src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1,5 +1,6 @@
 package redis.clients.jedis;
 
+import static redis.clients.jedis.Protocol.Keyword.WITHVALUES;
```

Comment:
This is a chance to sort the imports :)

Reply:
Should we avoid `import .*`?

Reply:
No.

`*` is avoided for Keywords because of conflicts with Commands. You are most welcome to use `*` resolving those conflicts.